lib/printer.ts

```diff
@@ -2487,26 +2487,29 @@ function genericPrintNoParens(path: any, options: any, print: any) {
 
       parts.push(path.call(print, "id"));
 
-      if (n.body && n.body.type === "TSModuleDeclaration") {
+      if (n.body) {
+        parts.push(" ");
         parts.push(path.call(print, "body"));
-      } else if (n.body) {
-        const bodyLines = path.call(print, "body");
-        if (bodyLines.isEmpty()) {
-          parts.push(" {}");
-        } else {
-          parts.push(" {\n", bodyLines.indent(options.tabWidth), "\n}");
-        }
       }
 
       return concat(parts);
     }
 
-    case "TSModuleBlock":
-      return path.call(
+    case "TSModuleBlock": {
+      const naked = path.call(
```

Comment:
Seems like `contents` would be a better name?

Reply:
Named it this because that's what is already done for `BlockStatement`: https://github.com/benjamn/recast/blob/aec4d901fafc195d4b5885ee91822e338df033a9/lib/printer.ts#L629